cmd/influx_tsm/main.go

```diff
@@ -170,11 +170,13 @@ func main() {
 	// Dump summary of what is about to happen.
 	fmt.Println("b1 and bz1 shard conversion.")
 	fmt.Println("-----------------------------------")
-	fmt.Println("Data directory is:       ", opts.DataPath)
-	fmt.Println("Backup directory is:     ", opts.BackupPath)
-	fmt.Println("Databases specified:     ", allDBs(opts.DBs))
-	fmt.Println("Database backups enabled:", yesno(!opts.SkipBackup), badUser)
-	fmt.Println("Parallel mode enabled:   ", yesno(opts.Parallel), runtime.GOMAXPROCS(0))
+	fmt.Println("Data directory is:                 ", opts.DataPath)
+	if !opts.SkipBackup {
+		fmt.Println("Backup directory is:               ", opts.BackupPath)
+	}
+	fmt.Println("Databases specified:               ", allDBs(opts.DBs))
+	fmt.Println("Database backups enabled:          ", yesno(!opts.SkipBackup), badUser)
+	fmt.Printf("Parallel mode enabled (GOMAXPROCS): %s (%d)", yesno(opts.Parallel), runtime.GOMAXPROCS(0))
```

Comment:
Nit: if the original extra line (I guess because other output follows after this?) is important, then there needs to be a `\n` added to this `Printf`.